packages/cursorless-vscode-e2e/suite/fixtures/recorded/languages/perl/changeKey.yml

```diff
@@ -0,0 +1,30 @@
+languageId: perl
```

Comment:
Nice. Worth also testing "key" when your cursor is in the value

Reply:
That's now fixed. It doesn't work if the cursor is on the comma that separates pairs, I'm not sure it is supposed to (I didn't see any e.g. ruby tests that test that), and if it is, I have no idea yet how to fix it.

Reply:
Yeah that's a symptom of https://github.com/cursorless-dev/cursorless/issues/436. It should get fixed automatically when we tackle that